checker/src/org/checkerframework/checker/index/lowerbound/LowerBoundAnnotatedTypeFactory.java

```diff
@@ -148,34 +150,19 @@ public void addComputedTypeAnnotations(Tree tree, AnnotatedTypeMirror type, bool
         }
     }
 
+    /** Returns the value checker's annotated type factory. */
     public ValueAnnotatedTypeFactory getValueAnnotatedTypeFactory() {
         return getTypeFactoryOfSubchecker(ValueChecker.class);
     }
 
+    /** Returns the minlen checker's annotated type factory. */
     public MinLenAnnotatedTypeFactory getMinLenAnnotatedTypeFactory() {
         return getTypeFactoryOfSubchecker(MinLenChecker.class);
     }
 
-    /**
-     * Either returns the exact value of the given tree according to the Constant Value Checker, or
-     * null if the exact value is not known. This method should only be used by clients who need
-     * exactly one value -- such as the binary operator rules -- and not by those that need to know
-     * whether a valueType belongs to an LBC qualifier. Clients needing a qualifier should use
-     * getLowerBoundAnnotationFromValueType instead of this method.
-     */
-    public Long getExactValueOrNullFromTree(Tree tree) {
-        AnnotatedTypeMirror valueType = getValueAnnotatedTypeFactory().getAnnotatedType(tree);
-        List<Long> possibleValues = possibleValuesFromValueType(valueType);
-        if (possibleValues != null && possibleValues.size() == 1) {
-            return possibleValues.get(0);
-        } else {
-            return null;
-        }
-    }
-
     /** Returns the type in the lower bound hierarchy that a Value Checker type corresponds to. */
     private AnnotationMirror getLowerBoundAnnotationFromValueType(AnnotatedTypeMirror valueType) {
-        // In the code, AnnotationMirror is abbr. as anm.
+        // In this code, AnnotationMirror is abbr. as anm.
```

Comment:
Cut the comment, because:
 * it is clear from context, and
 * the abbreviation is used only a method call name, not in variables declared in this method body.